cmd/kubeadm/app/cmd/reset.go

```diff
@@ -221,224 +240,14 @@ func (r *resetData) IgnorePreflightErrors() sets.String {
-	`)
-	fmt.Print(msg)
-
-	return nil
-}
-
-func getEtcdDataDir(manifestPath string, cfg *kubeadmapi.InitConfiguration) (string, error) {
-	const etcdVolumeName = "etcd-data"
-	var dataDir string
-
-	if cfg != nil && cfg.Etcd.Local != nil {
-		return cfg.Etcd.Local.DataDir, nil
-	}
-	klog.Warningln("[reset] No kubeadm config, using etcd pod spec to get data directory")
-
-	etcdPod, err := utilstaticpod.ReadStaticPodFromDisk(manifestPath)
-	if err != nil {
-		return "", err
-	}
-
-	for _, volumeMount := range etcdPod.Spec.Volumes {
-		if volumeMount.Name == etcdVolumeName {
-			dataDir = volumeMount.HostPath.Path
-			break
-		}
-	}
-	if dataDir == "" {
-		return dataDir, errors.New("invalid etcd pod manifest")
-	}
-	return dataDir, nil
-}
-
-func removeContainers(execer utilsexec.Interface, criSocketPath string) error {
-	containerRuntime, err := utilruntime.NewContainerRuntime(execer, criSocketPath)
-	if err != nil {
-		return err
-	}
-	containers, err := containerRuntime.ListKubeContainers()
-	if err != nil {
-		return err
-	}
-	return containerRuntime.RemoveContainers(containers)
-}
-
-// cleanDir removes everything in a directory, but not the directory itself
-func cleanDir(filePath string) error {
-	// If the directory doesn't even exist there's nothing to do, and we do
-	// not consider this an error
-	if _, err := os.Stat(filePath); os.IsNotExist(err) {
-		return nil
-	}
-
-	d, err := os.Open(filePath)
-	if err != nil {
-		return err
-	}
-	defer d.Close()
-	names, err := d.Readdirnames(-1)
-	if err != nil {
-		return err
-	}
-	for _, name := range names {
-		if err = os.RemoveAll(filepath.Join(filePath, name)); err != nil {
-			return err
-		}
-	}
-	return nil
-}
-
-// resetConfigDir is used to cleanup the files kubeadm writes in /etc/kubernetes/.
-func resetConfigDir(configPathDir, pkiPathDir string) {
-	dirsToClean := []string{
-		filepath.Join(configPathDir, kubeadmconstants.ManifestsSubDirName),
-		pkiPathDir,
-	}
-	fmt.Printf("[reset] Deleting contents of config directories: %v\n", dirsToClean)
-	for _, dir := range dirsToClean {
-		if err := cleanDir(dir); err != nil {
-			klog.Errorf("[reset] Failed to remove directory: %q [%v]\n", dir, err)
-		}
-	}
-
-	filesToClean := []string{
-		filepath.Join(configPathDir, kubeadmconstants.AdminKubeConfigFileName),
-		filepath.Join(configPathDir, kubeadmconstants.KubeletKubeConfigFileName),
-		filepath.Join(configPathDir, kubeadmconstants.KubeletBootstrapKubeConfigFileName),
-		filepath.Join(configPathDir, kubeadmconstants.ControllerManagerKubeConfigFileName),
-		filepath.Join(configPathDir, kubeadmconstants.SchedulerKubeConfigFileName),
-	}
-	fmt.Printf("[reset] Deleting files: %v\n", filesToClean)
-	for _, path := range filesToClean {
-		if err := os.RemoveAll(path); err != nil {
-			klog.Errorf("[reset] Failed to remove file: %q [%v]\n", path, err)
-		}
-	}
+// CriSocketPath returns the criSocketPath.
+func (r *resetData) CriSocketPath() string {
```

Comment:
please change this to `CRISocketPath`, so that we don't have to file separate PR later.
sorry that i've missed it.

Reply:
done!